src/sentry/api/serializers/models/project.py

```diff
@@ -411,6 +412,7 @@ def serialize(self, obj, attrs, user) -> ProjectSerializerResponse:
             "firstEvent": obj.first_event,
             "firstTransactionEvent": bool(obj.flags.has_transactions),
             "hasSessions": bool(obj.flags.has_sessions),
+            "hasProfiles": bool(obj.flags.has_profiles),
```

Comment:
Not sure if I should feature flag this attribute or document it in the public API?

Reply:
If you don't see this being around for a long time, or being useful for userland integrations I don't think there is much value in documenting this.

Reply:
This value will probably stick around long term as part of the profiling onboarding experience. So eventually, we'll have to document this as part of the public API. I guess I can do this now rather than later.